specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -700,7 +695,9 @@ def verify_cell_kzg_proof_batch(row_commitments_bytes: Sequence[Bytes48],
         assert len(proof_bytes) == BYTES_PER_PROOF
 
     # Get objects from bytes
+    row_commitments_bytes = list(set(commitments_bytes))
```

Comment:
Why not just call this deduplicated commitments?

Reply:
Hmm I like that better. I'll change this. Thanks.